dynamiqs/integrators/_utils.py

```diff
@@ -21,7 +21,7 @@
 from .core.abstract_integrator import AbstractIntegrator
 
 
-def _astimearray(x: QArrayLike | TimeArray) -> TimeArray:
+def astimearray(x: QArrayLike | TimeArray) -> TimeArray:
```

Comment:
We should really do these kind of renamings in the `main` branch, we're going to live with both `dev-qarray` and `main` for at least a month, so we want them to stay as close as can be. 😊